README.md

```diff
@@ -112,6 +115,8 @@ module.exports = {
     env: {
       ROOT_URL: 'http://app.com',
       MONGO_URL: 'mongodb://localhost/meteor'
+      PORT: 80, // (optional) defaults to 80
+      BIND: '0.0.0.0' // (optional) defaults to 0.0.0.0
```

Comment:
`env` are variables available to the Meteor app. `PORT` and `BIND` should be moved to the `docker` config.